fdbserver/storageserver.actor.cpp

```diff
@@ -6257,6 +6262,9 @@ ACTOR Future<Void> fetchKeys(StorageServer* data, AddingShard* shard) {
 			data->changeFeedRemovals.erase(fetchKeysID);
 		}
 		if (e.code() == error_code_actor_cancelled && !data->shuttingDown && shard->phase >= AddingShard::Fetching) {
+			if (SERVER_KNOBS->SS_BACKUP_KEYS_OP_LOGS && keys.intersects(backupLogKeys)) {
```

Comment:
Does adding fetchVersion helps?

Reply:
There are different versions that could be useful at this moment, I added several.